website/docs/d/filesystem.html.md

```diff
@@ -49,6 +36,8 @@ The `mount` block supports:
 
 * `options` - (Optional) Any additional options to be passed to the format-specific mkfs utility.
 
+* `path` - (Optional) The mount-point of the filesystem. A non-null entry indicates that the filesystem has already been mounted by the system at the specified path. This is really only useful for _/sysroot_.
```

Comment:
This language is mostly obsolete.  Borrowing the language from the [Ignition config spec](https://coreos.github.io/ignition/configuration-v3_3/):
```suggestion
* `path` - (Optional) The mount-point of the filesystem while Ignition is running relative to where the root filesystem will be mounted. This is not necessarily the same as where it should be mounted in the real root, but it is encouraged to make it the same.
```

Reply:
Updated accordingly